### ISSUE TITLE
Xcode 15 problem

### ISSUE THREAD

**Issue by nikitkabu** [SwiftGen/SwiftGen#1077]
After update to Xcode 15 I can't build the project, on Xcode 14 everything was fine:

/usr/bin/sandbox-exec -p "(version 1)
(deny default)
(import \"system.sb\")
(allow file-read*)
(allow process*)
(allow file-write*
    (subpath \"/private/tmp\")
    (subpath \"/private/var/folders/s8/zw78zq6n0w92cvn560gtprk80000gn/T\")
)
(deny file-write*
    (subpath \"/Users/nikitavishnevsky/Projects/polyMap/Help\")
)
(allow file-write*
    (subpath \"/Users/nikitavishnevsky/Library/Developer/Xcode/DerivedData/mapPolyTest-fuhdibhpuqxhlggyhqvisukeayvq/SourcePackages/plugins/help.output/Help/SwiftGenPlugin\")
)
" /Users/nikitavishnevsky/Library/Developer/Xcode/DerivedData/mapPolyTest-fuhdibhpuqxhlggyhqvisukeayvq/SourcePackages/artifacts/swiftgenplugin/swiftgen/swiftgen.artifactbundle/swiftgen/bin/swiftgen config run --verbose --config /Users/nikitavishnevsky/Projects/polyMap/Help/swiftgen.yml

Error: You don’t have permission to save the file “L10n.swift” in the folder “Help”.
Executing configuration file /Users/nikitavishnevsky/Projects/polyMap/Help/swiftgen.yml
 $ swiftgen strings --templatePath strings.stencil --param enumName=Localization --output ./Sources/Help/L10n.swift ../Resources/Localizations/en.lproj

What shall I do?

**Comment by yjdv** [comment]
I had the same problem if you haven't commit your changes revert your changes and don't let Xcode 15 to update your project. there is some weird configuration on Xcode 15 that have no access to swift plugins I tried running like a plugin running on brew and nothing worked have to revert the changes and build from that

**Comment by TomaszLizer** [comment]
Stumbled on same "issue" yesterday...
This is expected behaviour and actually it was bug of Xcode 14 that it worked this way and allowed writing to package directory.
Package plugins can only write to sandbox directory and it is fixed in Xcode 15 :)
Full story: https://github.com/SwiftGen/SwiftGen/issues/1075
**TLDR:** 
Read plugin docs: https://github.com/SwiftGen/SwiftGenPlugin#add-a-swiftgen-config
😅

**Comment by Liquidsoul** [comment]
This seems resolved by @TomaszLizer comment. I'll close this, reopen it if you still have the issue.
